site/content/in-dev/unreleased/metastores.md

```diff
@@ -32,6 +32,29 @@ In order to add other JDBC drivers, you have to build Polaris using the `eclipse
 
 ## Polaris Server Configuration
 
+### Relational JDBC
+Configure the `polaris.persistence` section in your Polaris configuration file
+(`application.properties`) as follows:
```

Comment:
As far as this PR goes we should probably use either env. vars of `application.properties` in all changes sections.

For unchanged sections, I think it's ok to defer the unification of configuration recommendations.

Reply:
Oops i missed it, apologies i fixed it for only the commented sections.